Bio/SeqIO/structure_io.py

```diff
@@ -0,0 +1,90 @@
+"""
+Base function for sequence parsers that read structures Bio.PDB parsers.
```

Comment:
Please remove the new line to follow PEP257 here
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings